data/access.log

```diff
@@ -0,0 +1,4 @@
+tp 4
```

Comment:
Shouldn't we use a better test case here? This would give zero output in the end. I'm thinking anything minimal that produces at least 2 co-clicked results.